altos-rust/altos-core/libs/heap/free_list_allocator/src/free_list.rs

```diff
@@ -153,7 +155,7 @@ impl FreeList {
     // Traverses the free list, looking for two sequential free blocks which return true
     // when passed into match_condition.
     fn find_block<F: Fn(Option<&BlockHeader>, &BlockHeader) -> bool>(&mut self, match_condition: F)
-        -> (Option<&'static mut BlockHeader>, Option<&'static mut BlockHeader>) {
+    -> (Option<&'static mut BlockHeader>, Option<&'static mut BlockHeader>) {
```

Comment:
I think we just missed this when we were fixing the indentation in all the files. The auto-indent seemed to like to put the two lines at the same indentation level.